wordpress-mobile/MediaPicker-iOS
# Add title to asset preview using the date of the asset.

To test:
 - Run the demo app
 - Open the full screen picker
 - LongPress/Force Touch an asset
 - Check that the tile of the asset on the asset preview is showing the date of the asset.
 - The format should Date\nTime for dates more than a week in distance and Weekday\nTime for dates that are less than a week in distance.
